app/views/historic_appointments/_historic_appointment.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="govuk-grid-column-one-quarter">
```

Comment:
We should re-use [this template](https://github.com/alphagov/collections/blob/main/app/views/shared/_historical_person.html.erb) here instead